extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceBuildTimeConfig.java

```diff
@@ -33,4 +23,22 @@ public static final FlywayDataSourceBuildTimeConfig defaultConfig() {
      */
     @ConfigItem(defaultValue = DEFAULT_LOCATION)
     public List<String> locations;
+
+    /**
+     * Comma-separated list of fully qualified class names of Callback implementations
+     * to use to hook into the Flyway lifecycle.
+     */
+    @ConfigItem
+    public Optional<List<String>> callbacks = Optional.empty();
```

Comment:
We also need to mention that for this to work, the class needs to have a no-args constructor.
Furthermore these classes must also not have any fields that hold state (unless that state is initialized in the constructor).